docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Migration from the previous Api resources
+type: Details
+---
+
+The migration is done automatically by a job. During the migration the old Api object is being migrated to an APIRule object, what may cause temporary unavailability of the exposed service. After the successful migration the old resource is removed. The migration might be skipped for some Api configuration, in which case the manual migration process may be done. It won't break the existing service exposure, but any changes to the Api object won't be effective.
+
+List remaining Apis to check whether any Api migration has been skipped
+
+```shell script
+kubectl get apis --all-namespaces
+```
+
+As Api resources are removed after a successful migration, all the resources that are left should be considered as not migrated. To get more details fetch logs from the migrator job
+
+```shell script
+kubectl logs api-gateway-api-migrator-job -n kyma-system
+```
+
+You will find logs saying which Api was not migrated and a reason for that.
+
+If the Api was not migrated due to the invalid status or a blacklisted label, you either shouldn't or will not be able to do the migration.
```

Comment:
Now this part here is begging for a header. Without a proper introduction (details on the migration job, see my previous comment), it's just out of context. My position is that you should spell it out to the user: the migration is a part of Kyma upgrade. You can verify your APIs once the upgrade process is over.